index.yaml

```diff
@@ -10930,6 +10930,24 @@ entries:
     - assets/rancher-prometheus-adapter/rancher-prometheus-adapter-2.12.101.tgz
     version: 2.12.101
   rancher-pushprox:
+  - annotations:
+      catalog.cattle.io/hidden: "true"
+      catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.25.0-0'
```

Comment:
I guess this gets updated when `Chart.yaml` gets updated too?

Reply:
yes